simapp/test_helpers.go

```diff
@@ -443,12 +444,12 @@ func (ao EmptyAppOptions) Get(o string) interface{} {
 //
 // TODO: Instead of using the mint module account, which has the
 // permission of minting, create a "faucet" account. (@fdymylja)
-func FundAccount(app *SimApp, ctx sdk.Context, addr sdk.AccAddress, amounts sdk.Coins) error {
-	if err := app.BankKeeper.MintCoins(ctx, minttypes.ModuleName, amounts); err != nil {
+func FundAccount(bankKeeper bankkeeper.Keeper, ctx sdk.Context, addr sdk.AccAddress, amounts sdk.Coins) error {
```

Comment:
addressing leftover comments from https://github.com/cosmos/cosmos-sdk/pull/8473/#pullrequestreview-591284702 cc/ @robert-zaremba

Reply:
My main desire was to move this function away from the `simapp` package. This functions is used in tests.
https://github.com/cosmos/cosmos-sdk/pull/8473#pullrequestreview-589519082

Reply:
@robert-zaremba to `testutil` for instance?

Reply:
`x/bank/client/testutil` could also be a candidate, even though `FundAccount` is not really a client util.

Reply:
This has received enough approvals, so let's tackle https://github.com/cosmos/cosmos-sdk/pull/9315#issuecomment-841797203 as part of a separate issue: #9346

Reply:
yes, we could put it to `testutil`. So, as you prefer, we can do it here , or in other PR.